README.md

```diff
@@ -1,6 +1,6 @@
-# LFP PAY API
+# PPS PAY API
```

Comment:
Do you think we should replace with `Penalty Payment Service (PPS) API` in the places that say `PPS PAY API` in the README?

Reply:
Yeah, that's a good idea. Will update.